qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,17 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = params.get("check_items").split(" ")
+            for item in vulnerabilities:
+                h_out = process.getoutput(check_cmd % item).strip()
+                g_out = session.cmd_output(check_cmd % item).strip()
+                if h_out != g_out:
+                    test.fail("Guest %s are not equal to Host %s"
+                              % (h_out, g_out))
```

Comment:
This whole sentence would be weird when the real output is filled in.

Reply:
thanks and update, please help to check.